pyproject.toml

```diff
@@ -1,3 +1,154 @@
+[tool.poetry]
+name = "fedora_messaging"
+version = "3.4.1"
+description = "A set of tools for using Fedora's messaging infrastructure"
+authors = [
+    "Fedora Infrastructure <admin@fedoraproject.org>"
+]
+license = "GPLv2+"
+readme = "README.rst"
+repository = "https://github.com/fedora-infra/fedora-messaging"
+maintainers = ["Fedora Infrastructure Team"]
+classifiers = [
+        "Development Status :: 5 - Production/Stable",
+        "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+        "Operating System :: POSIX :: Linux",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+    ]
+keywords = ["fedora"]
+packages = [
+    { include = "fedora_messaging" },
+]
+include = [
+    { path = "docs", format = "sdist" },
+		{ path = "configs", format = "sdist"},
+		{ path = "*.service", format = "sdist"},
+		{ path = "config.toml.example", format = "sdist"},
+]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+blinker = [
+    {version = "<=1.5", python = ">=3.6, <3.7"},
+    {version = "^1.5", python = "^3.7"}
+]
+click = [
+    {version = "<=8.0.4", python = ">=3.6, <3.7"},
+    {version = "^8.1.3", python = "^3.7"}
+]
+crochet = "1.12.0"
+jsonschema = [
+    {version = "<=3.2.0", python = ">=3.6, <3.8"},
+    {version = "^4.17.3", python = "^3.8"}
+]
+pika = [
+    {version = "<=1.3.1", python = ">=3.6, <3.7"},
+    {version = "^1.2.1", python = "^3.7"}
```

Comment:
This looks a bit weird, the version for Python 3.6 seems to be allowed to go higher than the minimum version for 3.7+. Could you check please?

Reply:
Ah, yes, here I had to go lower with the version, because in fedora38 there is the 1.2.1 and didn't fix it for python36 then.